Dockerfile_bis

```diff
@@ -0,0 +1,20 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-alpine3.10
```

Comment:
Why this dockerfile bis?

Reply:
oh my bad, I pushed something I didn't plan on pushing! I'll remove it!